textacy/datasets/wikipedia.py

```diff
@@ -51,6 +51,13 @@
     'de': 'Kategorie:'
 }
 
+PROJECTS = {
```

Comment:
Are the project name aliases purely for developer convenience, or are they "official" in some sense? If the former, I think I'd prefer simplicity and just use the `"wiki"` and `"wikinews"` names.

Reply:
the aliases are non-official and simply for convenience.  I can add a commit which removes them if you prefer

Reply:
Yeah, I think that's best. It's nice to have that 1:1 link between project name and file name. Thank you!

Reply:
Yeah, my original change was exactly that, remove the aliases and convert the dict to a set.  However, when  creating error messages for 404s I thought it was better to implement as 
```
PROJECTS = {'wikipedia': 'wiki', 'wikinews': 'wikinews'}
PROJECTS_INV = {v, k for k, v in PROJECTS.items()}
```
and rather than have the default value for project be 'wiki', instead have it be 'wikipedia' but `self.project` would get set to 'wiki via the lookup in PROJECTS.  Then in the case of a 404 error I can report that `{self.lang} variant of {PROJECTS_INV[self.project]}` doesn't exist, basically I think `'foo' variant of 'wikipedia' doesn't exist` is kinder to the user than `'foo' variant of 'wiki'` doesn't exist.   I'll push out the commit and you can see what you think.